README.md

```diff
@@ -193,7 +195,7 @@ Development time and resources for PHPMailer are provided by [Smartmessages.net]
 Contributions are very welcome, whether in beer 🍺, T-shirts 👕, or cold, hard cash 💰. Sponsorship through GitHub is a simple and convenient way to say "thank you" to PHPMailer's maintainers and contributors – just click the "Sponsor" button [on the project page](https://github.com/PHPMailer/PHPMailer).
 
 ## Changelog
-See [changelog](changelog.md).
+See [changelog](changelog.md) for the list of changes to PHPMailer.
```

Comment:
I don't think that's worth saying? It's not offering a definition for "changelog".